ietfdata/datatracker.py

```diff
@@ -976,6 +993,39 @@ def submission(self, submission_uri: SubmissionURI) -> Optional[Submission]:
         return self._retrieve(submission_uri, Submission)
 
 
+    def submission_event(self, event_uri: SubmissionEventURI) -> Optional[SubmissionEvent]:
+        return self._retrieve(event_uri, SubmissionEvent)
+
+
+    def submission_events(self,
+                        since      : str = "1970-01-01T00:00:00",
+                        until      : str = "2038-01-19T03:14:07",
+                        by         : PersonURI            = None,
+                        submission : SubmissionURI        = None,
```

Comment:
Should these be `Optional[]` - there are `is not None` tests below?

Reply:
Yes; fixed in 6fa463a

Reply:
Thanks!